service/src/main/java/uk/gov/hmcts/reform/fpl/model/common/JudgeAndLegalAdvisor.java

```diff
@@ -35,4 +36,12 @@ public static JudgeAndLegalAdvisor from(final Judge allocatedJudge) {
         }
         return judgeAndLegalAdvisorBuilder.build();
     }
+
+    @JsonIgnore
+    public JudgeAndLegalAdvisor resetJudgeProperties(YesNo useAllocatedJudge) {
```

Comment:
don't think JsonIgnore is needed, also don't think there will be a situation where we want to pass 'No' into this, so parameter probably isn't needed either (we reset to clear the input fields, we wouldn't want to populate with 'No' and clear the fields because it would delete what the user inputted)